lib/validators/checklist_result_extractor.rb

```diff
@@ -115,14 +115,14 @@ def find_negated_code(nodes, template, valueset, code)
     def find_attribute_values(node, code, source_criteria)
       # xpath expressions with codes
       xpath_map = {
-        'ANATOMICAL_LOCATION_SITE' => "./cda:targetSiteCode[@code='#{code}']",
         'DIAGNOSIS' => "./cda:entryRelationship/cda:act[./cda:code[@code='29308-4']]/cda:entryRelationship
                        /cda:observation/cda:value[@code='#{code}']", 'ORDINAL' => "./cda:priorityCode[@code='#{code}']",
```

Comment:
ORDINAL also wants to look at ./cda:entryRelationship/cda:observation[./cda:code[@code='260870009']]/cda:value[@code='#{code}']